models/r1_rover/r1_rover.sdf

```diff
@@ -1,24 +1,24 @@
-<sdf version='1.6'>
+<sdf version='1.7'>
```

Comment:
This breaks gazebo9, since it is not a supported sdf version. Why was this necessary?